clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -162,6 +180,12 @@ public Short validate(Object item) {
             else
                 throw new SchemaException(item + " is not a Short.");
         }
+
+        @Override
+        public String documentation() {
+            return "Represents an integer between -2<sup>15</sup> and 2<sup>15</sup>-1 inclusive. " +
+                    "The values are encoded on two bytes using network byte order (big-endian).";
```

Comment:
"encoded on two bytes using network byte order" -> "encoded using two bytes in network byte order"?

Similar for the other integer types if you like the suggestion.